SQL_1311: Add custom connector documentation (#19)
mongodb/mongo-powerbi-connector: mongodb-atlas-sql-interface.md

3. Select **Database** from the categories on the left, select **MongoDB Atlas SQL**, and then select **Connect**.

2. In the MongoDB Atlas SQL window that appears provide:
   - The **MongoDB URI** you retrieved previously. Make sure that it does not contain your username and password or it will be rejected.

>    - Your federated **Database** name.

Reviewer: I believe in Power BI you must enter in the database (even though it is also included in the string)

3. Select **Database** from the categories on the left, select **MongoDB Atlas SQL**, and then select **Connect**.

4. If this is the first time you're connecting to MongoDB Atlas SQL connector, a third-party notice will be displayed. 

>    Select Don't warn me again with this connector if you don't want this message to be displayed again, 

Reviewer 2: ```suggestion
   Select **"Don't warn me again with this connector"** if you don't want this message to be displayed again.
```

>    and then select **Continue**. 

Reviewer 2: ```suggestion

   Select **Continue**. 
```

> 5. In the MongoDB Atlas SQL window that appears provide:

Reviewer 2: ```suggestion
5. In the MongoDB Atlas SQL window that appears, fill in the following values:
```

>    - The **MongoDB URI** you retrieved previously. Make sure that it does not contain your username and password or it 

Reviewer 2: ```suggestion
   - _Required_. The **MongoDB URI**. _Required_. 
   
      Use the MongoDB URI you obtained in step 1.  Make sure that it does not contain your username and password. URIs containing username and/or passwords will be rejected.
```

Author: I think you did not mean to have _required_ twice. I couldn't find how to edit your suggestion, but I made the change to mark it required 👍

     will be rejected.

>    - Your federated **Database** name.

Reviewer 2: ```suggestion
   -  Your federated **Database** name. _Required_
   
      Use the name of the federated database you obtained in step 1.
```

>    - An optional **SQL statement**.

Reviewer 2: ```suggestion
   - A **SQL statement**.  _Optional_
```
